baseband/vlbi_base/base.py

```diff
@@ -131,8 +134,11 @@ def get_time0(self):
     time0 = property(get_time0, None, None)
 
     @property
-    def current_time(self):
-        """Time of the current offset in file."""
+    def time(self):
+        """Time of the sample pointer's current offset in file.
+
+        See also `start_time` for the start time of the file.
```

Comment:
Maybe also refer to `stop_time`.  I think it can be "See also ``start_time`` and ``stop_time``." (Pretty self-explanatory)